leakcanary-android-sample/src/main/res/values/strings.xml

```diff
@@ -18,7 +18,9 @@
     xmlns:tools="http://schemas.android.com/tools"
     tools:ignore="MissingTranslation">
   <string name="app_name">Example App</string>
-  <string name="helper_text">Rotate the screen 5 times to create 5 leaks and trigger a foreground
-    heap dump. Or press back to cause a leak and trigger a background heap dump.
+  <string name="helper_text"> - Rotate the screen 5 times to create 5 leaks and trigger a foreground heap dump\n
+    - Press the button to recreate activity 5 times to create 5 leaks and trigger a foreground heap dump\n
+    - Press back to cause a leak and trigger a background heap dump
```

Comment:
Pressing back does not cause a leak, it changes the threshold and triggers the heap dump.

We need to explain that either rotating or pressing the button will create 1 leak. That's it. Then we also need to explain that after 5 leaks the analysis will trigger.

...

Or maybe we don't, this is a sample app that generally most people won't even see.

Reply:
I spent good 10 minutes thinking about properly wording it and still can't figure it out. I'll file an issue to work on this sample app text.

Reply:
On the second thought, take a look at new version..